components/server/src/prebuilds/bitbucket-app.ts

```diff
@@ -199,12 +201,13 @@ export class BitbucketApp {
      * @param webhookInstaller the user account known from the webhook installation
      * @returns a promise which resolves to a user account and an optional project.
      */
-    protected async findProjectAndOwner(
+    private async findProjectAndOwner(
         cloneURL: string,
         webhookInstaller: User,
     ): Promise<{ user: User; project?: Project }> {
         try {
-            const project = await this.projectDB.findProjectByCloneUrl(cloneURL);
+            const projects = await this.projectService.findProjectsByCloneUrl(webhookInstaller.id, cloneURL);
+            const project = projects[0];
```

Comment:
Because webhook events are not yet suited for working on multiple projects we only build the first one. We want to remove webhooks soon and replace it with a simpler approach.

Reply:
We could alternatively simply iterate over all projects and run the logic, but this will overwrite the projectId and prebuildId in the WebhookEvent object we store in the DB.